### PR TITLE
Fix(cli): the problem of isListablePackage condition

### PR DESCRIPTION
**Pull request by mino01x**

### CODE REVIEW

File: packages/cli/src/commands/add/isListablePackage.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export function isListablePackage(config: Config, packageJson: PackageJSON) {
     return false;
   }
 
-  if (!config.privatePackages && packageJson.private) {
+  if (!config.privatePackages?.version && packageJson.private) {
```
Comment (Andarist) [inline packages/cli/src/commands/add/isListablePackage.ts:11]: Could you add a test case that would cover the fix?

Isn't `config.privatePackages` always an object? If yes, shouldn't it be this?
```suggestion
  if (!config.privatePackages.version && packageJson.private) {
```

Comment (mino01x) [reply packages/cli/src/commands/add/isListablePackage.ts]: i have added test case, is that ok?

**Review by JakeGinnivan — APPROVED** [review]
lgtm

**Comment by JakeGinnivan** [comment]
Just needs a changeset

File: packages/cli/src/commands/add/__tests__/add.ts [hunk 2]
```diff
@@ -230,4 +230,25 @@ describe("Changesets", () => {
     const { choices } = askCheckboxPlus.mock.calls[0][1][0];
     expect(choices).toEqual(["pkg-a", "pkg-c"]);
   });
+
+  it("should not include private packages with a version in the prompt", async () => {
```
Comment (Andarist) [inline packages/cli/src/commands/add/__tests__/add.ts:234]:
```suggestion
  it("should not include private packages with a version in the prompt if private packages are configured to be not versionable", async () => {
```

**Pull request merged**